docs/trace/links-creation-with-new-activities/README.md

```diff
@@ -0,0 +1,119 @@
+# Creating new root activities that link to an existing activity: An Example
+
+This example shows how to create new root activities that link to an existing
+activity. This can be useful in a fan-out or batched operation situation when
```

Comment:
a bit more background on what situation calls for this approach vs parenting everything to same root, would help future readers without background/context...